build-system/tasks/e2e/index.js

```diff
@@ -20,12 +20,13 @@ const gulp = require('gulp-help')(require('gulp'));
 const mocha = require('gulp-mocha');
 
 function e2e() {
-  return gulp.src(['test/e2e/*.js'], {read: false})
+  return gulp.src([
+    'test/e2e/*.js',
+    'extensions/**/test-e2e/*.js',
+  ], {read: false})
```

Comment:
Elevate this to a constant called `e2eTestPaths` in `build-system/config.js`

Reply:
Done!